patches/next-mdx-remote+4.0.3.patch

```diff
@@ -0,0 +1,14 @@
+diff --git a/node_modules/next-mdx-remote/index.d.ts b/node_modules/next-mdx-remote/index.d.ts
+index 9247c2a..09e9c9b 100644
+--- a/node_modules/next-mdx-remote/index.d.ts
++++ b/node_modules/next-mdx-remote/index.d.ts
+@@ -1 +1 @@
+-export * from './dist/index'
++export * from './dist/index.js'
```

Comment:
Here and in a couple of other patches: I had to add extensions for TS imports to resolve